util/manually-added-ecqm-data.json

```diff
@@ -0,0 +1,329 @@
+    "strata": [
+      {
+        "name": "screenedForCervicalCancer",
+        "description": "Women with one or more screenings for cervical cancer. Appropriate screenings are defined by any one of the following criteria:\r\n\r\n- Cervical cytology performed during the measurement period or the two years prior to the measurement period for women who are at least 21 years old at the time of the test\r\n \r\n- Cervical cytology/human papillomavirus (HPV) co-testing performed during the measurement period or the four years prior to the measurement period for women who are at least 30 years old at the time of the test"
+      }
+    ]
+  },
+  {
+    "eMeasureId": "CMS128v5",
+    "strata": [
+      {
+        "name": "continuousTreatmentAtLeast84Days",
+        "description": "Patients who have received antidepressant medication for at least 84 days (12 weeks) of continuous treatment during the 114-day period following the Index Prescription Start Date"
+      },
+      {
+        "name": "continuousTreatmentAtLeast180Days",
+        "description": "Patients who have received antidepressant medications for at least 180 days (6 months) of continuous treatment during the 231-day period following the Index Prescription Start Date"
+      }
+    ],
+    "overallAlgorithm": "simpleAverage"
+  },
+  {
+    "eMeasureId": "CMS136v6",
+    "strata": [
+      {
+        "name": "hadPractitionerVisit",
+        "description": "Patients who had at least one face-to-face visit with a practitioner with prescribing authority within 30 days after the IPSD."
+      },
+      {
+        "name": "hadPractitionerVisitAndFollowups",
+        "description": "Patients who had at least one face-to-face visit with a practitioner with prescribing authority during the Initiation Phase, and at least two follow-up visits during the Continuation and Maintenance Phase. One of the two visits during the Continuation and Maintenance Phase may be a telephone visit with a practitioner."
+      }
+    ],
+    "overallAlgorithm": "simpleAverage"
+  },
+  {
+    "eMeasureId": "CMS137v5",
+    "strata": [
+      {
+        "name": "treatmentWithin14Days",
+        "description": "Patients who initiated treatment within 14 days of the diagnosis"
+      },
+      {
+        "name": "treatmentPlusServicesWithin30",
+        "description": "Patients who initiated treatment and who had two or more additional services with an AOD diagnosis within 30 days of the initiation visit"
+      }
+    ],
+    "overallAlgorithm": "simpleAverage"
+  },
+  {
+    "eMeasureId": "CMS139v5",
+    "strata": [
+      {
+        "name": "screenedForFallRisk",
+        "description": "Patients who were screened for future fall risk at least once within the measurement period"
+      }
+    ]
+  },
+  {
+    "eMeasureId": "CMS145v5",
+    "overallAlgorithm": "weightedAverage",
+    "metricType": "multiPerformanceRate",
+    "strata": [
+      {
+        "description": "Beta Blocker Therapy for LVSD",
```

Comment:
Yeah, my descriptions were way too slim. I took a deeper look and produced these:

> Patients with at least 2 encounters with eligible professionals and left ventricular systolic dysfunction (LVEF <40%) who are prescribed beta-blocker therapy for LVSD. For these patients, beta-blocker therapy includes the following: bisoprolol, carvedilol, or sustained release metoprolol succinate.

and 

> Patients with at least 2 encounters with eligible professionals with a prior (resolved within last 3 years) myocardial infarction who are prescribed beta-blocker therapy. For patients with prior MI, beta-blocker therapy includes any agent within the beta-blocker drug class.